modules/services/control_service.lua

```diff
@@ -0,0 +1,101 @@
+require('atf.util')
+
+local functionId = require('function_id')
+local json = require('json')
+local constants = require('protocol_handler/ford_protocol_constants')
+local events = require('events')
+local Event = events.Event
+local expectations = require('expectations')
+
+
+local Expectation = expectations.Expectation
+local SUCCESS = expectations.SUCCESS
+local FAILED = expectations.FAILED
+
+
+local module = {}
+local mt = { __index = { } }
+
+function module.Service(session)
+  local res = { }
+  res.session = session
+  setmetatable(res, mt)
+  return res
+end
+
+function mt.__index:Send(message)
+  message.frameType = constants.FRAME_TYPE.CONTROL_FRAME    
+  self.session:Send(message)
+  xmlReporter.AddMessage("mobile_connection","Send",{message})    
+  return message
+end
+
+function mt.__index:StartService(service)
+	xmlReporter.AddMessage("StartService", service)
+  local startSession =
+  {
+    frameType = 0,
+    serviceType = service,
+    frameInfo = 1,
+    sessionId = self.session.sessionId.get(),
+  }
+  self:Send(startSession)
```

Comment:
@VVeremjova I support that it is not best Idea to create expectations after StartSession Sent